source/instructors/authoring/guides/custom_button.rst

```diff
@@ -26,20 +26,21 @@ e.g.
 
 To launch a process
 *******************
-This will launch a process and execute a terminal command.
+This will launch a process and execute a terminal command. In this example a Python program is run and output is displayed in the guide.
 
 .. code:: markdown
 
-    {Button Text}(command paramater1 parameter2 parameterN)
```

Comment:
I think removing this will cause more confusion, can't expect everyone to ONLY want to execute one command.
Perhaps add practical example below